### DISCUSSION TITLE
How to change de ac_mode parameter without editing the yaml file

### DISCUSSION THREAD

**Discussion by juanjhdez** [ScratMan/HASmartThermostat#72] [Q&A]
Hi all,
I'd like to know if there is a way to modify the ac_mode parameter without editing the yaml file. It is because I have an underfloor system, it uses hot water in winter for heating and cold water in summer for cooling, but it is the same circuit and same electric valves. I want to automatize in HA the change summer mode to winter mode and viceversa.
Thanks in advance for your help!

**Comment by soyxan** [comment]
Same request here. I would like to have a general switch in HA that I can use to change between winter/summer mode.

**Comment by ScratMan** [comment]
Hello, latest [2023.7.0-beta2](https://github.com/ScratMan/HASmartThermostat/releases/tag/2023.7.0-beta2) allows switching between heating and cooling without restarting HA. Please test.
